ui/app/adapters/pki-config.js

```diff
@@ -69,22 +69,24 @@ export default ApplicationAdapter.extend({
     return hash({
       backend: backendPath,
       id: this.id(backendPath),
-      der: this.rawRequest(derURL, 'GET', { unauthenticated: true }).then(response => response.blob()),
-      pem: this.rawRequest(pemURL, 'GET', { unauthenticated: true }).then(response => response.text()),
-      ca_chain: this.rawRequest(chainURL, 'GET', { unauthenticated: true }).then(response => response.text()),
+      der: this.rawRequest(derURL, 'GET', { unauthenticated: true }).then((response) => response.blob()),
+      pem: this.rawRequest(pemURL, 'GET', { unauthenticated: true }).then((response) => response.text()),
+      ca_chain: this.rawRequest(chainURL, 'GET', { unauthenticated: true }).then((response) =>
+        response.text()
```

Comment:
just curious but did your editor force this to be on one line because of length?